arviz/plots/ecdfplot.py

```diff
@@ -46,26 +52,41 @@ def plot_ecdf(
     values : array-like
         Values to plot from an unknown continuous or discrete distribution.
     values2 : array-like, optional
-        Values to compare to the original sample.
+        deprecated: values to compare to the original sample. Instead use
+        `cdf=scipy.stats.ecdf(values2).cdf.evaluate`.
     cdf : callable, optional
         Cumulative distribution function of the distribution to compare the original sample.
         The function must take as input a numpy array of draws from the distribution.
     difference : bool, default False
         If True then plot ECDF-difference plot otherwise ECDF plot.
-    pit : bool, default False
-        If True plots the ECDF or ECDF-diff of PIT of sample.
-    confidence_bands : bool, default None
-        If True plots the simultaneous or pointwise confidence bands with `1 - fpr`
-        confidence level.
-    pointwise : bool, default False
-        If True plots pointwise confidence bands otherwise simultaneous bands.
+    confidence_bands : str or bool, optional
+        - False: No confidence bands are plotted.
+        - "pointwise": Compute the pointwise (i.e. marginal) confidence band.
+        - True or "simulated": Use Monte Carlo simulation to estimate a simultaneous confidence
```

Comment:
In the next PR I will add deterministic bands, which will become the default here.